app/controllers/proposals_controller.rb

```diff
@@ -24,6 +24,14 @@ class ProposalsController < ApplicationController
   helper_method :resource_model, :resource_name
   respond_to :html, :js
 
+  def index
+    super
+    if params[:advanced_search].try(:[], :geozone_id).present? && params[:order].blank?
+      @current_order = "oldest"
```

Comment:
Why do we want to order by oldest when searching by geozone but by hot score when using a different filter :thinking:? From the UX perspective I find it very confusing. I'm in the "Most active" tab, I filter by geozone and then I'm in the "Oldest" tab, then I clear the geozone filter and I'm in the "Most active" tab again. It feels like I'm not in control of what I'm trying to do.

Reply:
Yes, I agree. They asked explicitly for this behaviour, but maybe if we explain the inconvenience, they would agree with us.